ftw/publisher/core/adapters/dx_field_data.py

```diff
@@ -151,8 +165,21 @@ def unpack(self, name, field, value):
                 schema.interfaces.ITime,
                 schema.interfaces.IDatetime,
                 ]):
-            if value:
-                return DateTime.DateTime(value).asdatetime()
+            if value and value.get('date_or_time'):
+                # Expect ISO-8601 datetime/time/date where zone was stripped (not recalculated)
+                date_or_time = dateutil.parser.parse(value['date_or_time'])
+                zone_str = value.get('zone')
+
+                if self._provided_by_one_of(field, [schema.interfaces.IDate]):
```

Comment:
me tooo!

Reply:
Bonus points for renaming the method to `_provides_one_of` :-)

Reply:
I won't rename the method because it don't think it belongs into this PR but I did the #metoo